lib/url.js

```diff
@@ -23,19 +23,19 @@
 
 const { toASCII } = process.binding('config').hasIntl ?
   process.binding('icu') : require('punycode');
+
 const { hexTable } = require('internal/querystring');
-const internalUrl = require('internal/url');
-exports.parse = urlParse;
-exports.resolve = urlResolve;
-exports.resolveObject = urlResolveObject;
-exports.format = urlFormat;
-exports.URL = internalUrl.URL;
-exports.URLSearchParams = internalUrl.URLSearchParams;
-exports.domainToASCII = internalUrl.domainToASCII;
-exports.domainToUnicode = internalUrl.domainToUnicode;
 
+// WHATWG URL implementation provided by internal/url
+const {
+  URL,
+  URLSearchParams,
+  domainToASCII,
+  domainToUnicode,
+  formatSymbol
+} = require('internal/url');
 
-exports.Url = Url;
+// Legacy url.parse() API
```

Comment:
Given we don't have an alternative for relative URLs yet, I'm not sure we should call it legacy in the source since there are use cases whatwg-url doesn't cover.